pkg/api/v1/types.go

```diff
@@ -640,12 +640,8 @@ type Container struct {
 	Lifecycle      *Lifecycle           `json:"lifecycle,omitempty" description:"actions that the management system should take in response to container lifecycle events; cannot be updated"`
 	// Optional: Defaults to /dev/termination-log
 	TerminationMessagePath string `json:"terminationMessagePath,omitempty" description:"path at which the file to which the container's termination message will be written is mounted into the container's filesystem; message written is intended to be brief final status, such as an assertion failure message; defaults to /dev/termination-log; cannot be updated"`
-	// Deprecated - see SecurityContext.  Optional: Default to false.
```

Comment:
v1 is still completely changeable. It hasn't been finalized yet and is not exposed by default.